Day-6-7/src/components/UserCardInfo.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router";
+import { fetchUserData } from "../utils/fetchData";
+
+const UserCardInfo = () => {
+  const { id } = useParams();
+  const [userData, setUserData] = useState([]);
+  const [userRepo, setuserRepo] = useState([]);
+  console.log(id);
+  useEffect(() => {
+    displayUserInfo();
+  }, [id]);
+
+  const displayUserInfo = async () => {
+    const url = `https://api.github.com/users/${id}`;
+    const userInfo = await fetchUserData(url);
+    console.log(userInfo);
+    getUserRepo(userInfo.repos_url);
+    setUserData([userInfo]);
+  };
+
+  const getUserRepo = async (url) => {
+    const repoData = await fetchUserData(url);
+    console.log(repoData);
+    repoData.sort(function (a, b) {
+      return b.stargazers_count - a.stargazers_count;
+    });
+    setuserRepo(repoData.slice(0, 5));
+  };
+
+  console.log(userRepo);
+
+  if (userRepo && userRepo.length === 0) {
+    return <div className="loader">Loading....</div>;
+  } else {
+    return (
+      <div className="user-container">
```

Comment:
Start using section, articles etc. semantic html, is a good habit when you will write SSR applications using reactjs.